Code/Tools/ProjectManager/Source/PythonBindings.cpp

```diff
@@ -866,7 +866,8 @@ namespace O3DE::ProjectManager
             try
             {
                 projectInfo.m_projectName = Py_To_String(projectData["project_name"]);
-                projectInfo.m_displayName = Py_To_String_Optional(projectData, "display_name", projectInfo.m_projectName);
+                // set the project display name to a length of 22 characters if the name is longer than that
+                projectInfo.SetProjectDisplayName(Py_To_String_Optional(projectData, " display_name ", projectInfo.m_projectName));
```

Comment:
The dictionary key we get from Python shouldn't include these spaces
```suggestion
                projectInfo.SetProjectDisplayName(Py_To_String_Optional(projectData, "display_name", projectInfo.m_projectName));
```